superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -41,6 +43,7 @@ interface CRUDCollectionProps {
   ) => ReactNode)[];
   onChange?: (arg0: any) => void;
   tableColumns: Array<any>;
+  scrollTable?: boolean;
```

Comment:
nit: maybe name this `stickyHeader`?

Reply:
This is a behaviour that is specific to modals using tables. I am worried that `stickyHeader` might not convey the actual functionality. Something on the line `tableMode` or `stickyTableMode`?

Reply:
`stikyHeader` is a pretty popular pattern for large tables and not limited to modals, even within Superset. We've seen it used in SQL Lab results preview, Table viz, and Explore data preview... to name few.

Material UI has a similar API: https://material-ui.com/zh/api/table/

Reply:
Oh I didn't know that. That's great to know! I'll change it. Thanks a lot @ktmud!